tool/src/kotlin/mod.rs

```diff
@@ -309,10 +309,7 @@ impl<'a, 'cx> TyGenContext<'a, 'cx> {
             .expect("Failed to render opaque return block")
     }
 
-    const WRITEABLE_RETURN: &'static str = r#"
-val returnString = DW.writeableToString(writeable)
-DW.lib.diplomat_buffer_writeable_destroy(writeable)
-return returnString"#;
+    const WRITEABLE_RETURN: &'static str = "\nreturn DW.writeableToString(writeable)";
```

Comment:
Question: where did you remove the call to `destroy`?

Based on the generated code, it looks like it just moved somewhere else.

Reply:
It's in `writeableToString`, in a `finally` block